Week1/homework/js-exercises/4-logString.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var myString ='Jalal Al Attar';
```

Comment:
Do you think it's better to use `let` or `var` here :)?

Reply:
yes because if I used **CONST** I will show this a beautiful  **SyntaxError**
SyntaxError: Identifier 'myString' has already been declared.